### ISSUE TITLE
Failed to compile

### ISSUE THREAD

**Issue by skanga** [Th3Wall/Fakeflix#16]
Failed to compile on Windows 10

c:\bin>git clone https://github.com/Th3Wall/Fakeflix
Cloning into 'Fakeflix'...
remote: Enumerating objects: 2740, done.
remote: Counting objects: 100% (125/125), done.
remote: Compressing objects: 100% (47/47), done.
remote: Total 2740 (delta 100), reused 86 (delta 78), pack-reused 2615
Receiving objects: 100% (2740/2740), 1.22 MiB | 2.54 MiB/s, done.
Resolving deltas: 100% (1749/1749), done.

c:\bin>cd fakeflix

c:\bin\Fakeflix>npm install

> re2@1.15.9 install c:\bin\Fakeflix\node_modules\re2
> install-from-cache --artifact build/Release/re2.node --host-var RE2_DOWNLOAD_MIRROR || npm run rebuild

Trying https://github.com/uhop/node-re2/releases/download/1.15.9/win32-x64-72.br ...
Writing to build/Release/re2.node ...
Done.

> husky@4.3.8 install c:\bin\Fakeflix\node_modules\husky
> node husky install

husky > Setting up git hooks
husky > Done

> node-sass@5.0.0 install c:\bin\Fakeflix\node_modules\node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v5.0.0/win32-x64-72_binding.node
Download complete  ] - :
Binary saved to c:\bin\Fakeflix\node_modules\node-sass\vendor\win32-x64-72\binding.node
Caching binary to C:\Users\skanga\AppData\Roaming\npm-cache\node-sass\5.0.0\win32-x64-72_binding.node

> core-js@3.6.5 postinstall c:\bin\Fakeflix\node_modules\@firebase\polyfill\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> core-js@2.6.12 postinstall c:\bin\Fakeflix\node_modules\babel-runtime\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> core-js@3.9.0 postinstall c:\bin\Fakeflix\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> core-js-pure@3.9.0 postinstall c:\bin\Fakeflix\node_modules\core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

> ejs@2.7.4 postinstall c:\bin\Fakeflix\node_modules\ejs
> node ./postinstall.js

Thank you for installing EJS: built with the Jake JavaScript build tool (https://jakejs.com/)

> protobufjs@6.10.2 postinstall c:\bin\Fakeflix\node_modules\protobufjs
> node scripts/postinstall

> husky@4.3.8 postinstall c:\bin\Fakeflix\node_modules\husky
> opencollective-postinstall || exit 0

Thank you for using husky!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/husky/donate

> node-sass@5.0.0 postinstall c:\bin\Fakeflix\node_modules\node-sass
> node scripts/build.js

Binary found at c:\bin\Fakeflix\node_modules\node-sass\vendor\win32-x64-72\binding.node
Testing binary
Binary is fine

> swiper@6.5.0 postinstall c:\bin\Fakeflix\node_modules\swiper
> node postinstall.js

 Love Swiper? Support Vladimir's work by donating or pledging on patreon:
 > On Patreon https://patreon.com/vladimirkharlampidi
 > On Open Collective https://opencollective.com/swiper
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\webpack-dev-server\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 2460 packages from 978 contributors and audited 2469 packages in 152.055s

151 packages are looking for funding
  run `npm fund` for details

found 230 vulnerabilities (1 low, 219 moderate, 9 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

c:\bin\Fakeflix>npm start

> fakeflix@0.1.0 start c:\bin\Fakeflix
> react-scripts start

i ｢wds｣: Project is running at http://172.17.32.1/
i ｢wds｣: webpack output is served from
i ｢wds｣: Content not from webpack is served from c:\bin\Fakeflix\public
i ｢wds｣: 404s will fallback to /
Starting the development server...
Failed to compile.

./node_modules/framer-motion/dist/es/render/index.js
Attempted import error: '__spreadArray' is not exported from 'tslib'.

**Comment by Th3Wall** [comment]
Hi @skanga, which version of node are you using?

**Comment by skanga** [comment]
node v12.18.4

**Comment by kkanekii** [comment]
hey i know this may not work but try adding a .git when you clone it, for eg:
``` git clone https://github.com/Th3Wall/Fakeflix.git```
this is very basic but it might be the reason

**Comment by ranfysvalle02** [comment]
@skanga - I encountered this issue as well. In my particular case it was because I was running npm install from a Macbook Air 2020 M1; node-sass is deprecated and doesn't work in M1 (https://stackoverflow.com/questions/65594119/node-sass-usage-is-deprecated-and-will-be-removed-in-a-future-major-version)

To fix the issue, I just replaced 

"node-sass": "^5.0.0",

with 

"sass": "^1.36.0",

at package.json; 

Removed ./node_modules, and ran npm install.

Hope that helps anyone else running into this issue.
